hack/arktos-up.sh

```diff
@@ -554,6 +559,13 @@ if [[ "${START_MODE}" != "nokubelet" ]]; then
     esac
 fi
 
+# Applying mizar cni
+if [[ "${CNIPLUGIN}" == "mizar" ]]; then
+  ${KUBECTL} --kubeconfig="${CERT_DIR}/admin.kubeconfig" create configmap system-source --namespace=kube-system --from-literal=name=arktos --from-literal=company=futurewei
+  # ${KUBECTL} --kubeconfig="${CERT_DIR}/admin.kubeconfig" apply -f ${KUBE_ROOT}/hack/testdata/mizar/deploy.mizar.next.yaml 
+  ${KUBECTL} --kubeconfig="${CERT_DIR}/admin.kubeconfig" apply -f https://raw.githubusercontent.com/CentaurusInfra/mizar/dev-next/etc/deploy/deploy.mizar.dev.yaml
```

Comment:
URL to the yaml file need to be updated to Mizar release

Reply:
Test with Mizar and default arktos-up both tested. Revert arktos-up changes for mizar. It will be fully tested and merged in a different PR.